[MRG+1] Update about.rst

Naereen opened the pull request:
The link `Kemal Eren`_ was linked to nothing.

I tried adding one link for some Google Summer of Code contributors (to a LinkedIn, GitHub account or so), but it should be their choice, not an arbitrary choice.

Naereen committed:
Update about.rst

`Kemal Eren`_ linked to nothing.

agramfort commented:
LGTM

ogrisel commented on doc/about.rst:
```diff
@@ -93,10 +93,10 @@ to work on scikit-learn through the
+- 2011 - `Vlad Niculae <http://vene.ro/>`_.
+- 2012 - `Vlad Niculae <http://vene.ro/>`_, `Immanuel Bayer <http://www.informatik.uni-konstanz.de/rendle/members/bayer0/>`_.
+- 2013 - `Kemal Eren <http://www.kemaleren.com/>`_, `Nicolas Trésegnie <https://www.linkedin.com/in/nicolastr>`_.
 - 2014 - Hamzeh Alsalhi, Issam Laradji, Maheshakya Wijewardena, Manoj Kumar.
```
Some of those links (e.g. Vlad) are already defined at the bottom of `whats_new.rst`: there is no need to repeat them here as they already work:

http://scikit-learn.org/dev/about.html

Naereen replied:
OK. Then only the link for Kemal Eren is not defined, sorry I should have only added his.
![kemal_eren](https://cloud.githubusercontent.com/assets/11994719/13257435/01523296-da4f-11e5-9b59-6c2b726008e9.png)

ogrisel commented:
@Naereen could you update the PR and move the URL for Kemal Eren at the end of the whats_new.rst file instead? (and the remove the other redundant links you added to this PR).

Naereen committed:
Update about.rst

Remove useless links, they will be added in ``whats_new.rst``.

Naereen committed:
Update whats_new.rst

Added 2 links (David Cournapeau, Immanuel Bayer); and uniformying a little bit the "people" lists (using same indent and spaces).

Naereen commented on doc/about.rst:
```diff
@@ -93,10 +93,10 @@ to work on scikit-learn through the
-- 2011 - `Vlad Niculae`_
-- 2012 - `Vlad Niculae`_, Immanuel Bayer.
-- 2013 - `Kemal Eren`_, Nicolas Trésegnie
+- 2007 - `David Cournapeau`_.
```
I removed the inlined links and added them to the end of ``whats_new.rst``.

Naereen commented on doc/about.rst:
```diff
@@ -175,17 +175,14 @@ The 2013' Paris international sprint
    :width: 120pt
    :target: http://www.telecom-paristech.fr/
 
-
```
Useless empty lines.

Naereen commented on doc/whats_new.rst:
```diff
@@ -514,7 +514,7 @@ Bug fixes
       that affected :class:`linear_model.DictionaryLearning` and
       :class:`covariance.GraphLasso`. (`#5337 <https://github.com/scikit-learn/scikit-learn/pull/5337>`_)
-      By `Oliver Grisel`_.
+      By `Olivier Grisel`_.
```
Typo on his name.

Naereen commented on doc/whats_new.rst:
```diff
@@ -1569,168 +1569,168 @@ People
-*   1	Andrew Ash
-*   1	Pietro Zambelli
-*   1	staubda
+ * 312  Olivier Grisel
```
Just spacing uniformization.

Naereen commented on doc/whats_new.rst:
```diff
@@ -2457,7 +2458,7 @@ List of contributors for release 0.13 by number of commits.
  *  10  Kyle Beauchamp
  *   9  Brian Cheung
- *   9  Immanuel Bayer
+ *   9  `Immanuel Bayer`_
```
Added a link for him.

Naereen commented on doc/whats_new.rst:
```diff
@@ -2536,14 +2537,14 @@ Changelog
- *   1  `Alexandre Gramfort`_
- *   1  `Gilles Louppe`_
- *   1  `Mathieu Blondel`_
+ * 14  `Peter Prettenhofer`_
```
Same, spaces more uniform across the file.

Naereen commented on doc/whats_new.rst:
```diff
@@ -2727,7 +2729,7 @@ People
  *   1  Charles-Pierre Astolfi
  *   1  Dan O'Huiginn
- *   1  David Cournapeau
+ *   1  `David Cournapeau`_
```
And also added a link to him.

Naereen commented on doc/whats_new.rst:
```diff
@@ -3657,32 +3658,19 @@ People
 People that made this release possible preceded by number of commits:
 
-   * 207  `Olivier Grisel`_
-
```
Removed the useless empty lines here.

Naereen commented on doc/whats_new.rst:
```diff
@@ -4063,19 +4051,33 @@ David Huard, Dave Morrill, Ed Schofield, Travis Oliphant, Pearu Peterson.
 .. _Vighnesh Birodkar: https://github.com/vighneshbirodkar
 
 .. _Chyi-Kwei Yau: https://github.com/chyikwei
+
```
Added one empty line between every one in this list (more consistent).

Naereen commented:
@ogrisel Done.
I also cleaned up a little bit the ``whats_new.rst`` file (a few typo, adding these two links, uniforming the spaces and indent for "people" lists ..).

tguillemot commented on doc/whats_new.rst:
```diff
@@ -1569,168 +1569,168 @@ People
+ *   1  Andrew Ash
+ *   1  Pietro Zambelli
+ *   1  staubda
 
```
Could it be interesting to unify the links to the people ? 
For example Olivier Grisel is not linked here.

Naereen replied:
I would.
That's what I wanted to do at first, but it required too much work.

TomDLT replied:
You can play with [this](https://gist.github.com/TomDLT/1645afb6a0b2cb713416d1740d96cc4b) as a first step.

TomDLT commented:
LGTM
Can you rebase on top of master and resolve your conflicts?

Naereen commented:
Hi, thanks for your script and suggestions. I am on leave, and will take care of this asap when I come back.

amueller commented:
do you want to rebase?

Naereen commented:
I don't think it's needed.

amueller commented:
@Naereen yes it is needed, otherwise I can't merge. See the github message at the bottom of the PR.

Naereen commented:
OK. I guess I am not skilled enough with git cli, I tried for 2 hours to rebase on master and do what you asked, but I failed.

I won't have time to try that again for such small changes - sorry.

amueller commented:
@Naereen that's too bad. It's a good skill to learn though ;)
http://docs.scipy.org/doc/numpy/dev/gitwash/development_workflow.html#rebasing-on-master
